backend/server/api/controllers/todo/controller.js

```diff
@@ -1,13 +1,32 @@
-const db = require("../../services/todos.service");
+import ToDoService from "../../services/todos.service";
 
 export class Controller {
   stub(req, res) {
-    console.log(`Temporal... ${db}...`);
+    //	console.log(`Temporal... ${db}...`);
     return res.json(req.body);
   }
   create(req, res) {
-    console.log(`Temporal...${req} ${res}...`);
-    return true;
+    const todo = {
+      name: req.body.name,
+      description: req.body.description,
+      isDone: false,
+      dueDate: req.body.dueDate,
+    };
+    if (todo.description == undefined) {
+      //If no description was provided, then set it equal to an empty string
+      //so the data base accepts an empty description
+      todo.description = "";
+    }
+    if (todo.name == undefined) {
```

Comment:
You're checking for `name` and `description` but what happens if the user doesn't send you `dueDate` and/or `isDone`? You should have validations for that.

Reply:
Done